### PR TITLE
MUIC-436

### PR DESCRIPTION
**Pull request by mustafa-ogun-ozturk**

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java [hunk 1]
```diff
@@ -739,6 +753,12 @@ private void initViews() {
         newMessagesCountBadgeView = view.findViewById(R.id.new_messages_badge_view);
         addAttachmentMenu = view.findViewById(R.id.add_attachment_menu);
         attachmentsRecyclerView = view.findViewById(R.id.add_attachment_queue);
+        operatorTypingAnimation = view.findViewById(R.id.operator_typing_animation_view);
+        operatorTypingAnimation.addValueCallback(
+                new KeyPath("**"),
+                LottieProperty.COLOR_FILTER,
+                frameInfo -> new PorterDuffColorFilter(MaterialColors.getColor(this, R.attr.gliaBaseNormalColor), PorterDuff.Mode.SRC_ATOP) // TODO what color should be put?
```
Comment (deniss-mc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java:760]: use gliabrandprimary from this.theme

**Review by deniss-mc — CHANGES_REQUESTED** [review]

File: widgetssdk/src/main/res/layout/chat_view.xml [hunk 2]
```diff
@@ -72,11 +72,27 @@
...
+        app:lottie_rawRes="@raw/chat_typing_indicator"
+        app:lottie_scale="0.5"
+        tools:visibility="visible" />
+
     <View
         android:id="@+id/divider_view"
         android:layout_width="match_parent"
         android:layout_height="1dp"
-        android:layout_marginTop="@dimen/glia_medium"
+        android:layout_marginTop="@dimen/glia_xx_large"
```
Comment (deniss-mc) [inline widgetssdk/src/main/res/layout/chat_view.xml:95]: not sure if this is desired

**Review by deniss-mc — APPROVED** [review]
One comment otherwise looks good

File: widgetssdk/src/main/res/layout/chat_view.xml (hunk 2 above)
Comment (mustafa-ogun-ozturk) [reply widgetssdk/src/main/res/layout/chat_view.xml:95]: That is how it looks on Figma

**Pull request merged**